doctr/models/recognition/utils.py

```diff
@@ -0,0 +1,85 @@
+# Copyright (C) 2021, Mindee.
+
+# This program is licensed under the Apache License version 2.
+# See LICENSE or go to <https://www.apache.org/licenses/LICENSE-2.0.txt> for full license details.
+
+from typing import List
+from Levenshtein import distance
+
+
+__all__ = ['merge_sequences', 'merge_multi_sequences']
+
+
+def merge_sequences(a: str, b: str, dil_factor: float) -> str:
+    """Compute the (best) overlap between 2 character sequences and merge them.
+
+    Args:
+        a: first char seq, suffix should be close to b's prefix.
+        b: second char seq, prefix should be close to a's suffix.
+        dil_factor: dilation factor of the boxes to overlap, should be > 1. This parameter is
+        only used when the mother sequence is splitted on a character repetition
```

Comment:
you need to indent that by 4 spaces for the documentation to render correctly ;)